### PR TITLE
Ingress status publish service

### PR DESCRIPTION
**Pull request by ijsnellf**
Should enable the ingress status syncer to obtain the load balancer IP address.

### CODE REVIEW

File: platform/kube/ingressstatus.go [hunk 1]
```diff
@@ -57,11 +59,16 @@ func NewIngressStatusSyncer(mesh *config.ProxyMeshConfig, client *Client,
 		&v1beta1.Ingress{}, options.ResyncPeriod, cache.Indexers{},
 	)
 
+	var publishService string
+	if mesh.IngressService != "" {
```
Comment (kyessenov) [inline platform/kube/ingressstatus.go]: i think it's better not to run status syncer when the field is not defined, WDYT?

Comment (ijsnellf) [inline platform/kube/ingressstatus.go]: If this field isn't defined the status syncer will update the ingress resource with the node IP. This is somewhat useful if you are using a NodePort setup, as in Minikube. Though in that case you'll still need to use the 30000+ port.

**Comment by ijsnellf** [comment]
jenkins rebuild manager/presubmit

File: platform/kube/ingressstatus.go (hunk 1 above)
Comment (kyessenov) [inline platform/kube/ingressstatus.go]: ok, fair enough.

**Comment by ijsnellf** [comment]
Contributes to #214.

**Pull request merged**